addons/packages/fluxcd-source-controller/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  name: fluxcd.source.controller.community.tanzu.vmware.com
+spec:
+  displayName: "Flux Source Controller"
+  longDescription: "The source-controller is a Kubernetes operator, specialised in artifacts acquisition from external sources such as Git, Helm repositories and S3 buckets."
+  shortDescription: "The source-controller is a Kubernetes operator, specialised in artifacts acquisition from external sources such as Git, Helm repositories and S3 buckets."
```

Comment:
Long description and short description is same here. Can you update the short description?